packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/repository.ts

```diff
@@ -389,6 +391,7 @@ export class SavedObjectsRepository implements ISavedObjectsRepository {
       migrationVersion,
       coreMigrationVersion,
       typeMigrationVersion,
+      managed: setManaged({ optionsManaged: managed }),
```

Comment:
nit: I think this would be simpler

```suggestion
      managed: managed ?? false,
```

Reply:
I prefer _not_ to embed a default/fallback in the code itself and rather use the utility for that. Using the utility _everywhere_ has the benefit of if the logic ever changes, we only need to make the change in one place.